roottest/root/io/perf/slowreading/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+
+ROOTTEST_ADD_TEST(files
+                  COMMAND ${CMAKE_COMMAND} -E echo "copy files"
+                  COPY_TO_BUILDDIR rootfiles/DST_60.root
+                                   rootfiles/DST_61.root
+                                   rootfiles/DST_62.root
+                  FIXTURES_SETUP root-io-perf-slowreading-files-fixture)
+
+foreach(macro 60 61 62)
+  ROOTTEST_COMPILE_MACRO(NuEvent_${macro}.cxx
+                         FIXTURES_SETUP root-io-perf-slowreading-${macro}-fixture)
+
+  if(macro EQUAL 60)
+    # measure reading time and write file
+    ROOTTEST_ADD_TEST(mark
+                      MACRO Read.C
+                      MACROARG "\"NuEvent_60_cxx\", \"DST_60.root\", 1"
+                      FIXTURES_REQUIRED root-io-perf-slowreading-files-fixture
+                                        root-io-perf-slowreading-60-fixture
+                      FIXTURES_SETUP root-io-perf-slowreading-mark-fixture)
+  endif()
+
+  foreach(file 60 61 62)
+    ROOTTEST_ADD_TEST(check-macro${macro}-file${file}
+                      MACRO Read.C
```

Comment:
In the Makefile version (via the trytwice), it was allowing for one of two run to be too slow before failing ...

Reply:
But why twice is better than once?
Files must be in cache.
If test will fail too often - one can increase margins.

Reply:
Yes this is the other option.  However the net effect is different.  If one increase the margin, it allows for more performance degradation before noticing.   So twice was thought at the time to be better that once because it allowed to ignore more fluctuation (including unexpected disk and/or cache usage due to other usage at the time of the run) while still keeping a lower margin.

For now, let's at least add a comment mentioning that a previous incarnation of the test(s) was running the test a second time if the first try was slower than expected.

Reply:
It is not clear to me how to repeat test twice without failing when first time was too slow.
In principle, one can do it inside macro - if we really want it.

I will add comments

Reply:
> It is not clear to me how to repeat test twice without failing when first time was too slow.

[Just a comment, no action needed] a simple way would have been to keep using the `trytwice.sh` script :)

Reply:
Any usage of shell scripts excludes Windows - that we do not want